feat: internationalization Polish

uiopak opened the pull request:
Provides login, messages, and console in Polish.
Adds missing language references to fr, it and zh where other languages are used.
Makes consistent translation of languages in Chinese translation. (don't translate languages names)

- [x] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [x] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

fforootd commented:
Hi @uiopak 

Thank you for this PR! 🚀 

let me get somebody looking into this PR

CC @hifabienne

eliobischof reviewed (commented):
I added some translations using [DeepL](https://www.deepl.com/translator) that were missing because we added new features in the mean time.

Would you mind checking the commit @1feef32f17e5dd55d5546af593f0c91a1f1f503c?

@zitadel/engineers could you please check if you have unmerged keys that need to be translated as well?

uiopak commented on console/src/assets/i18n/pl.json:
```diff
@@ -0,0 +1,1957 @@
+    "SETAVAILABLE": "ustaw jako dostępne",
+    "SETUNAVAILABLE": "ustaw jako niedostępne",
+    "CONFIG": "Konfiguracja",
+    "STATE": "Stan",
```
State is translated to stan.
Status is also status in Polish.
The difference between both words and their meaning is the same as in English.

uiopak commented:
I corrected the new translations and added missing keys that I found in https://github.com/zitadel/zitadel/blob/add-quotas/internal/static/i18n/en.yaml when I was checking English version.

uiopak commented:
I corrected the new translation, added missing key that I found in https://github.com/zitadel/zitadel/blob/add-quotas/internal/static/i18n/en.yaml and corrected some earlier translations.

uiopak commented:
I corrected the new translation, checked and fixed indentation in yaml files

eliobischof commented:
Hey @uiopak, we are very happy about your contribution and for making ZITADEL more accessible.

We would like to show our appreciation by sending you some swag. If you are interested, please send us a mail, referring to this PR to hi@zitadel.com.
